Dockerfile

```diff
@@ -3,7 +3,7 @@
 ARG fmriprep_version=25.0.0
 
 FROM condaforge/miniforge3 AS conda
-RUN conda config --system --append channels https://fsl.fmrib.ox.ac.uk/fsldownloads/fslconda/public && \
+RUN conda config --system --append channels https://fsl.fmrib.ox.ac.uk/fsldownloads/fslconda/public/ && \
```

Comment:
The build should be fixed now, so it should be fine to leave this out. By the way, I think adding a slash here does nothing but trigger a re-build of everything (since the Docker layer cache is invalidated).

Reply:
Ok, thanks for the explanation👍